docs/design/autoware-interfaces/components/sensing.md

```diff
@@ -0,0 +1,52 @@
+# Sensing
+
+![Node diagram](images/Sensing-Bus-ODD-Architecture.drawio.svg)
+
+## Inputs
+
+### The sensing module contains the driver for the sensor, so there is no input
+
+## Output
+
+### Camera image
+
+Image data from camera. Used by the Perception.
+
+- [sensor_msgs/Image](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/Image.html)
+
+### Lidar original pointcloud
+
+Pointcloud without preprocessing. Used by the Perception and Localization.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
+
+### Lidar filtered pointcloud
+
+Pointcloud after preprocessing. Used by the Perception.
+
+- [sensor_msgs/PointCloud2](https://docs.ros.org/en/noetic/api/sensor_msgs/html/msg/PointCloud2.html)
```

Comment:
Which topic are you referring to with these two "LiDAR original pointcloud" and "LiDAR filtered pointcloud"? 

Localization and perception module also uses preprocessed pointcloud.

Reply:
"filtered pointcloud" means "preprocessed pointcloud", because there are various filters in the pointcloud_preprocessor.
We usually use /sensing/lidar/top/rectified/pointcloud as an initial pointcloud in perception module and subscribe directly to topics from lidar driver.
As to localization, we didn't pay much attention to the changes in the new version. I will fix it soon

Reply:
I think we can just say that perception and localization module uses "preprocessed pointcloud". Both module do not use raw pointcloud data.

Reply:
All done